moodler/submission.py

```diff
@@ -62,13 +62,16 @@ def needs_grading(self):
         Returns True if the submission needs grading.
         Does this by checking the grading status and the grading timestamp vs the last modification timestamp
         """
-        return ("notgraded" == self.gradingstatus) or self.resubmitted
+        return self.gradingstatus in ("notgraded", "notmarked") or self.resubmitted
```

Comment:
@maxandron requires this change as `notmarked` seems to be used instead `notgraded` status when marking workflow enabled.

Reply:
nice catch